Example/Info.plist

```diff
@@ -4,20 +4,12 @@
 <dict>
 	<key>AppCenterSecret</key>
 	<string>$(RMA_APP_CENTER_SECRET)</string>
-	<key>CFBundleDevelopmentRegion</key>
-	<string>en</string>
 	<key>CFBundleExecutable</key>
 	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIdentifier</key>
-	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
-	<key>CFBundleInfoDictionaryVersion</key>
-	<string>6.0</string>
-	<key>CFBundleName</key>
-	<string>$(PRODUCT_NAME)</string>
-	<key>CFBundlePackageType</key>
-	<string>APPL</string>
+	<string>com.rakuten.tech.mobile.miniapp.MiniAppDemo</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
```

Comment:
We need this to differentiate the DEV/STG and prod builds
MARKETING_VERSION = $(RMA_SDK_VERSION)$(RMA_DEMO_APP_BUILD_TYPE)

Reply:
@rleojoseph regarding plist it got an update and is now automatically removing these from `Info.plist` and putting them into the project file (https://github.com/rakutentech/ios-miniapp/blob/26c4cc05a4fd00788d9da247e386f9de833b3faf/MiniAppSPM/Sample%20SPM.xcodeproj/project.pbxproj#L1115) although some keys are missing I'll check